src/Admin/PageAdmin.php

```diff
@@ -62,11 +62,11 @@ public function __construct(
 
     protected function configureRoutes(RouteCollectionInterface $collection): void
     {
-        $collection->add('compose', '{id}/compose', [
-            'id' => null,
+        $collection->add('compose', $this->getRouterIdParameter().'/compose', [
+            $this->getIdParameter() => null,
         ]);
-        $collection->add('compose_container_show', 'compose/container/{id}', [
-            'id' => null,
+        $collection->add('compose_container_show', 'compose/container/'.$this->getRouterIdParameter(), [
+            $this->getIdParameter() => null,
```

Comment:
pedantic change, I saw it being used on SiteAdmin, I like it more that using id directly  wdyt @VincentLanglet ?

Reply:
Indeed seems nice